CHANGES.rst

```diff
@@ -7,6 +7,9 @@ Version 3.3.6
 
 *Unreleased*
 
+- Implement conditional fields (:issue:`1228`, :pr:`6678`,
+thanks :user:`omegak, unconventionaldotdev` and :user:`Moliholy, unconventionaldotdev`).
```

Comment:
I think something went wrong there... you have the credits twice, and it's not under "improvements"

Reply:
One is for @OmeGak and another one for me. I just moved it to the correct section.

Reply:
but did he contribute to it? in any case, you can put all 3 names in the same :user: block separated with commas

Reply:
> but did he contribute to it?

Yes, he did. I made the commits, but we worked together.

> in any case, you can put all 3 names in the same :user: block separated with commas

Done!